engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/em/BigIntegerEMOperator.java

```diff
@@ -58,22 +60,25 @@ public void accumulateCumulative(@NotNull RowSequence inputKeys,
                         handleBadData(this, true);
                     } else if (isNullTime) {
                         // no change to curVal and lastStamp
+                    } else if (curVal == null) {
+                        // If the data looks good, and we have a null computed value, accept the current value
+                        curVal = new BigDecimal(input, control.bigValueContextOrDefault());
+                        lastStamp = timestamp;
                     } else {
                         final BigDecimal decimalInput = new BigDecimal(input, control.bigValueContextOrDefault());
```

Comment:
Minor nitpick, you can move definition of decimalInput closer to line 80 where it is used after the if checks.